### PR TITLE
Setup

### LINKED ISSUE #43: Migration Setup

### PR DESCRIPTION
**Pull request by algibson1**
Created migrations, created rake tasks, setup seeds

### CODE REVIEW

**Review by pcbennett108 — APPROVED** [review]
It all works as expected. Great!

File: lib/tasks/csv_load.rake [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      customer.created_at = row[:created_at]
+      customer.updated_at = row[:updated_at]
+      customer.save
+    end
+    ActiveRecord::Base.connection.reset_pk_sequence!('customers')
+    puts "Customers Loaded"
+  end
+end
+
+namespace :csv_load do
```
Comment (brad2412) [inline lib/tasks/csv_load.rake:27]: Nice job!

**Review by brad2412 — APPROVED** [review]

File: lib/tasks/csv_load.rake [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  task :all => :environment do
+    all = [
+      :customers,
+      :invoices,
+      :merchants,
+      :items,
+      :invoice_items,
+      :transactions
+    ]
+    all.each do |file|
```
Comment (topher-nullset) [inline lib/tasks/csv_load.rake:140]: if it works, don't touch it...

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,12 @@
...
 #   Character.create(name: "Luke", movie: movies.first)
+
+Transaction.destroy_all
+InvoiceItem.destroy_all
+Invoice.destroy_all
+Item.destroy_all
+Merchant.destroy_all
+Customer.destroy_all
+
+Rake::Task['csv_load:all'].invoke
```
Comment (topher-nullset) [inline db/seeds.rb:16]: i really like how this works

**Review by topher-nullset — APPROVED** [review]
yay

**Pull request merged**